### PR TITLE
IACO-608 add readme

### PR DESCRIPTION
**Pull request by grozen**
This took quite a little longer than I anticipated so as to get run instructions that actually work (hopefully, it would be great if we can get some people around the office to try them out).

As always, you may want to see everything rendered, [here](https://github.com/yuval-netanel/klarna-on-demand-android/tree/feature/IACO-608-add-readme).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+# Klarna on Demand
 [![Build Status](https://travis-ci.org/yuval-netanel/klarna-on-demand-android.svg?branch=master)](https://travis-ci.org/yuval-netanel/klarna-on-demand-android)
+
+Klarna on-demand allows you to integrate Klarna's payment solution in mobile apps that offer on demand services. It's a perfect fit for apps selling concert tickets, taxi rides, food pick-ups, etc.
+
+This project contains Klarna's on-demand purchase SDK for Android 4+, as well as a sample application utilizing the SDK.
+
+![It's Klarna in your App](screenshot.png)
+
+While not necessary, the simplest way to get going with both the sample application and the SDK is to use [Gradle](https://www.gradle.org/) and so all the following instructions will focus on that approach. For an alternative approach, see the [official documentation (coming soon)](http://developers.klarna.com).
```
Comment (yuval-netanel) [inline README.md]: It's also very simple to download the jar and add it to the project classpath (once we have the jar we can add a link)
It is more right to use gradle **or maven**

File: README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+
+## Contributing
+You'd like to help us out? That's great! Here's what you need to do in order to contribute.
+
+### Prerequisites
+As per [running the application](#running_the_application).
+
+### Setup
+
+1. Fork the project and clone your repository
```
Comment (yuval-netanel) [inline README.md]: How you decide when to put a dot in the end of a bullet sentence? :)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+2. Import the project into a Gradle-friendly work environment such as [Android Studio](http://developer.android.com/sdk/index.html) or [Eclipse](https://eclipse.org/) with [Gradle tooling support](https://github.com/spring-projects/eclipse-integration-gradle/)
+3. Get busy
+
+### Tests
+Pull requests must include tests for their proposed fixes/functionality. We use [JUnit](http://junit.org/) to write our tests and you can find the tests themselves [here](klarna-on-demand/src/androidTest/java/com/klarna/ondemand).
```
Comment (yuval-netanel) [inline README.md]: @tomerlevi444 remember to change the link in your branch if you change the test dir path

**Comment by yuval-netanel** [comment]
Approved!

File: README.md (hunk 2 above)
Comment (grozen) [inline README.md]: Usually it has to do with whether or not I am forced to include multiple sentences in a single bullet.

**Pull request merged**